Client/Frontend/Browser/Tabs/RemoteTabs/RemoteTabsPanelState.swift

```diff
@@ -7,32 +7,44 @@ import Redux
 import Shared
 import Storage
 
+/// State for RemoteTabsPanel. WIP.
 struct RemoteTabsPanelState {
     let refreshState: RemoteTabsPanelRefreshState
     let clientAndTabs: [ClientAndTabs]
-    let allowsRefresh: Bool // True if hasSyncableAccount
-    let showingError: RemoteTabsPanelErrorState?
-    let syncIsSupported: Bool // Reference: `prefs.boolForKey(PrefsKeys.TabSyncEnabled)`
+    let allowsRefresh: Bool                                // True if `hasSyncableAccount()`
+    let showingEmptyState: RemoteTabsPanelEmptyState?      // If showing empty (or error) state
+    let syncIsSupported: Bool                              // Reference: `prefs.boolForKey(PrefsKeys.TabSyncEnabled)`
 
     static func emptyState() -> RemoteTabsPanelState {
         return RemoteTabsPanelState(refreshState: .loaded,
                                     clientAndTabs: [],
                                     allowsRefresh: false,
-                                    showingError: nil,
+                                    showingEmptyState: .noTabs,
                                     syncIsSupported: true)
     }
 }
 
+/// Status of tab refresh.
 enum RemoteTabsPanelRefreshState {
```

Comment:
nit: but we declared enum on the top of the class (I missed it in the previous PR)

Reply:
✅ Moved these to the top of the file (if you meant something different LMK)